src/testsuite/vec_f64_dummy.c

```diff
@@ -306,16 +316,16 @@ test_vf64_ibm128_vec (vf64_t lval)
 {
   return (vec_pack_longdouble (lval));
 }
-
+#ifndef __clang__
```

Comment:
why? add a comment.

Reply:
/* Disable for clange becuase does not support the asm constraints
   (%L) to access 2nd FRP of FPR pair. */